src/client.js

```diff
@@ -188,8 +283,17 @@ class TouchPortalClient extends EventEmitter {
     this.sendArray(connectorArray);
   }
 
+  /**
+   * updateActionData() allows you dynamically update action data field
+   * minValue and maxValue
+   * 
+   * @param {String} actionInstanceId - action instance Id
+   * @param {Object} data - an object that contains minValue, maxValue, id and type
+   * 
+   * @return {void}
+   */
   updateActionData(actionInstanceId, data) {
-    if (data.id === undefined || data.id === '' || data.minValue === undefined || data.minValue === '' || data.maxValue === undefined || data.maxValue === '' || data.type === undefined || data.type === '') {
+    if (!data.id || !data.minValue || !data.maxValue || !data.type) {
```

Comment:
Whoops actually this won't work for `minValue` and `maxValue` since they could be zero. Those could be turned into `typeof minValue != "number"`

Reply:
good catch...